### ISSUE TITLE
Don't download to "Complete" folder

### ISSUE THREAD

**Issue by forthrin** [transmission/transmission#671]
Request: Always move files back to the "Incomplete" folder when selecting more files for partially completed torrents.

How it works today:

1. Start a torrent
1. Select only *some* files to download
1. (When finished, the files are moved to the "Complete" folder)
1. Select some more files to download
1. (Download continues, to the "Complete" folder)

How it should work:

Same as above, but point 5: Files should be transferred back to the "Incomplete" folder before download resumes.

Reason: on macOS, Spotlight will constantly reindex partially downloaded files, which is meaningless. To avoid this you can set the "Incomplete" folder as "Do not index", but then you will still get the constant reindexing of the "Complete" folder if you resume downloads like described above. If you set the "Complete" folder as "Do not index", you can't use Spotlight to reach those files.

In conclusion: Always move files back to the "Incomplete" folder when selecting more files for partially completed torrents. It's more correct, and eliminates problems.

**Comment by forthrin** [comment]
Anyone? It's been a year since the issue was opened.

**Comment by forthrin** [comment]
Evidently, no-one is reading these issues.
